config.xml

```diff
@@ -20,7 +20,7 @@
     <icon src="res/icon.png" />
     <platform name="android">
         <preference name="android-minSdkVersion" value="19" />
-        <preference name="android-targetSdkVersion" value="27" />
+        <preference name="android-targetSdkVersion" value="28" />
```

Comment:
why?

Reply:
Cordova-android in the version 8.0.0 introduces support for Android API level 28. Si think we should take advantage of new APIs for more recent devices that use this app. Android devices with at least API level 19 are still supported.

Reply:
But if you think it's not necessary or I said something wrong, I think there's no problem reverting this change.

Reply:
I didn't see anything about "minimal version", if it's still 19 no problem.
I just wanted to make sure we don't drop support of not-very-fresh phones.